apps/protocol-frontend/src/components/MyAttestationsTable.tsx

```diff
@@ -86,21 +86,30 @@ const MyAttestationsTable = ({
         }) => {
           const status = getValue();
 
+          const guildHasVerificationFramework =
+            row.original.guilds[0].guild?.verification_setting_id !== null;
+          const attestationThreshold =
+            row.original.guilds[0].attestation_threshold;
+
+          const frameworkSettingThreshold =
+            row.original.guilds[0].guild.verification_setting
+              ?.num_of_attestations;
+          const pillUnverifiedLabel = `${attestationThreshold}/${frameworkSettingThreshold}`;
```

Comment:
these are usually gonna be the same (row.original.guilds[0].attestation_threshold; & row.original.guilds[0].guild.verification_setting?.num_of_attestations;) since the latter is copied onto the former each time the job runs. if you want # attestations / # attestations need for verified, you should get the number of current attestations on the contribution.

Reply:
Thanks! I think this may fix the issue I'm facing